sdk/core/core-http/test/defaultHttpClientTests.ts

```diff
@@ -69,6 +69,38 @@ describe("defaultHttpClient", function() {
       await promise;
       assert.fail("");
     } catch (err) {
+      err.name.should.be.equal("AbortError");
+      err.should.not.be.instanceof(AssertionError);
+    }
+  });
+
+  it("should allow canceling requests before request is made", async function() {
+    const resourceUrl = `/fileupload`;
+    httpMock.post(resourceUrl, async () => {
+      await sleep(10000);
```

Comment:
does this mean that this test will add 10 seconds to the test run time?

Reply:
Nope! Because the request gets cancelled.

Reply:
of course! Thanks :)